elteikthesis.cls

```diff
@@ -256,10 +264,11 @@
 \nottoggle{hypenation}{\RequirePackage[none]{hyphenat}}{}
 
 % Beállítás alternatív módon
-%\RequirePackage{setspace}
-%\singlespacing		% ~1.0 linespread
-%\onehalfspacing	% ~1.3 linespread
-%\doublespacing		% ~1.6 linespread
+\RequirePackage{setspace}
+\setstretch{1.5}
```

Comment:
The line spacing should be 1.3, as that is somewhat equivalent to 1.5 line spacing in MS Word. It seems it was not correct   so far.

Reply:
Acording to https://tex.stackexchange.com/questions/442030/setting-spaces-with-titlespacing-and-fonts 1.423 is the equialent of words 1.5 spacing. I checked with Libreoffice and with this setting matches the "1.5 lines" option.

Reply:
Somewhat funny how the definition of line spacing has so many possible interpretations. The linked reference mentions that line spacing in MS Word depends on the typeface/font and the value 1.427465 only applies for Times New Roman.

Anyway, I have checked and the formatting requirements at the faculty also do not provide any further information on the definition of line spacing, so technically both 1.5, 1.3 and 1.427465 can be valid :smile: